src/lib.rs

```diff
@@ -12,9 +12,10 @@
 //! =============
 //! If you get errors about SFML headers not being found, or linker errors, that probably means
 //! SFML is not installed in a global location.
-//! In that case, you can set two environment variables to help rust-sfml find the required files:
+//! In that case, you can set three environment variables to help rust-sfml find the required files:
```

Comment:
`LD_LIBRARY_PATH` is a linux/*nix specific environment variable to help the dynamic loader find the required dynamic libraries, so putting it here is misleading.

- It doesn't help `rust-sfml` find the required files. It has nothing to do with `rust-sfml`.
- It doesn't work on windows, this is linux/*nix speficic.

Also this technically isn't about linking at compile time, it's about loading dynamic libs at runtime.

Reply:
Awesome. Of course, the documentation on the main SFML is superior since it is SFML-specific, but allowing users to fix all their edge cases from one documentation source is more user-friendly. 

It's fixed now